### PR TITLE
refactor series reader to make it small

### PR DESCRIPTION
**Pull request by mychaow**
### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

File: server/src/main/java/org/apache/iotdb/db/query/reader/series/SeriesReader.java [hunk 1]
```diff
@@ -18,16 +18,6 @@
  */
 package org.apache.iotdb.db.query.reader.series;
```
Comment (jixuan1989) [inline server/src/main/java/org/apache/iotdb/db/query/reader/series/SeriesReader.java:20]: avoid more changes like this in the future.

**Comment by jixuan1989** [comment]
Why for this improvement?

**Comment by mychaow** [comment]
> Why for this improvement?

I think SeriesReader is too long,  move the utils class to separate class will make it looks more clear.

File: server/src/main/java/org/apache/iotdb/db/query/reader/series/SeriesReader.java (hunk 1 above)
Comment (mychaow) [reply server/src/main/java/org/apache/iotdb/db/query/reader/series/SeriesReader.java:20]: Ok

**Comment by Alima777** [comment]
Hi, From the perspective of OOP, TimeOrderUtils is only used in `SeriesReader` and should be better to be the inner class of it. There is no need to move it for the too long reason.

**Pull request closed without merging**